core/interfaces/status.py

```diff
@@ -56,7 +56,7 @@ class StatusInterface(ui.Interface, GitCommand):
       [u] unstage file                      [U] unstage all staged files
       [d] discard changes to file           [D] discard all unstaged changes
       [h] open file on remote
-      [M] launch external merge tool for conflict
+      [M] launch external merge tool
```

Comment:
Should we perhaps move the `[M]` to the `CONFLICTS` section too?

Reply:
I've never used an external merge tool. Is it useful to open unstaged file with merge tool? If the answer is yes, I guess we should put it here.

Reply:
Yes